src/skmultiflow/anomaly_detection/half_space_trees.py

```diff
@@ -516,11 +521,11 @@ def update_model(self, node):
 
         """
         if node is not None:
-            if node.r is not 0 or node.l is not 0:
-                node.r = node.l
+            if node.right_mass is not 0 or node.left_mass is not 0:
+                node.right_mass = node.left_mass
 
-            if node.l is not 0:
-                node.l = 0
+            if node.left_mass is not 0:
+                node.left_mass = 0
```

Comment:
When I run my the tests locally, pytest warns me that we should use something like: node.rigth_mass != 0 rather than relying on is not 0. What do you think? Could we leverage this PR to apply this fix?

Reply:
Nice catch. I changed it in cc37f898c0cb91ed786c23f9413fad11b9929592 as well as other non-standard uses of `is not`.

Reply:
Nice!